rlp/rlpgen/gen.go

```diff
@@ -35,6 +35,8 @@ type buildContext struct {
 	rawValueType *types.Named
 
 	typeToStructCache map[types.Type]*rlpstruct.Type
+
+	internalMethods bool
```

Comment:
Do we need this option actually? Can't it be always `true` such that generated decoder/encoder methods are always unexported?

Reply:
If they're unexported then they won't work. `EncodeRLP` and `DecodeRLP` are equivalent to `json.Marshaler` and `json.Unmarshaler` and in this case we just want the option to override them with a hook that will be introduced in the next PR.